packages/cactus-test-tooling/src/main/typescript/quorum/quorum-test-ledger.ts

```diff
@@ -105,6 +105,10 @@ export class QuorumTestLedger implements ITestLedger {
     const hostPort = await this.getRpcApiPublicPort();
     return `http://${ipAddress}:${hostPort}`;
   }
+  public async getRpcApiWsHost(): Promise<string> {
+    const ipAddress = "127.0.0.1";
+    return `ws://${ipAddress}:8546`;
```

Comment:
@rwat17 If it's not too much trouble please store the default port in a single place within this codefile and also make it queryable by other code via a method the same way `getRpcApiPublicPort()`  makes it possible for the HTTP port.

Making it this way will make the code less britle (resistant to changes around it) because we don't have to replace the port in multiple places if for whatever reason it has to be changed and so the probability of mistakes will be reduced.